### PR TITLE
goals on same level

### PR DESCRIPTION
**Pull request by kushagra189**
state
I have made the change of appearance of each goal in the form of horizontal containers.

### CODE REVIEW

File: about-us.html [hunk 1]
```diff
@@ -15,7 +15,7 @@
         <div class="row">
             <div class="col-md-12" >
                 <div class="row">
-                    <div class="col-md-8 col-md-push-2" > 
+                    <div class="col-md-8 col-md-push-2" >
```
Comment (quozl) [inline about-us.html]: Unintended whitespace change?

File: about-us.html [hunk 2]
```diff
@@ -48,13 +48,13 @@ <h2>MISSION</h2>
     <div class="container">
         <div class="row">
             <div class="col-md-12" >
-                <div class="row">  
+                <div class="row">
```
Comment (quozl) [inline about-us.html]: Unintended whitespace change?

File: about-us.html [hunk 3]
```diff
@@ -48,13 +48,13 @@ <h2>MISSION</h2>
...
                     <div class="col-md-8 col-md-push-2" >
                         <div class="section-title text-center">
                             <h2>PRINCIPLES</h2>
                             <p class="customParagraphStyle">In order for Sugar to be successful, it needs the participation of a large number of people who share common goals while maintaining independence, so that each participant has the ability to act independently. For these reasons, Sugar Labs subscribes to the principles described <a href="https://flors.wordpress.com/2008/05/04/the-paradigm-of-the-open-organization/" class="hrefCustomColor">here</a>, which are the author's own translation of an <a href="http://web.archive.org/web/20050317231119/http://interactors.coop/organizacionabierta" class="hrefCustomColor">original text in Spanish.</a></p>
                         </div>
-                    </div>   
+                    </div>
```
Comment (quozl) [inline about-us.html]: Unintended whitespace change?

File: about-us.html [hunk 4]
```diff
@@ -110,7 +110,7 @@ <h6>A special <a href="https://wiki.sugarlabs.org/go/Sugar_Labs/Thank_You" class
                 </div>
             </div>
         </div>
-    </div>    
+    </div>
```
Comment (quozl) [inline about-us.html]: Unintended whitespace change?

File: about-us.html [hunk 5]
```diff
@@ -123,11 +123,11 @@ <h6>A special <a href="https://wiki.sugarlabs.org/go/Sugar_Labs/Thank_You" class
                         <h2>GOALS</h2>
                     </div>
                 </div>
-                <div class="row">
-                    <div class="col-md-4">
-                        <div class="border-box boxDownload2">
+                <!-- <div class="row"> -->
```
Comment (quozl) [inline about-us.html]: Purpose of changing to comment?

File: about-us.html [hunk 6]
```diff
@@ -163,7 +163,7 @@ <h2 >Community Goals</h2>
                             </ul>
                         </div>
                     </div>
-                </div>
+                <!-- </div> -->
```
Comment (quozl) [inline about-us.html]: Purpose of changing to comment?

File: about-us.html [hunk 7]
```diff
@@ -201,11 +201,11 @@ <h2>Projects</h2>
...
                         <ul>
                             <li><a href="http://co.sugarlabs.org/go/Laboratorios_Azúcar_Colombia" class="hrefCustomColor">Sugar Labs Colombia</a></li>
-                            <li><a href="http://cl.sugarlabs.org/go/Página_Principal" class="hrefCustomColor">Sugar Labs Chile</a></li> 
+                            <li><a href="http://cl.sugarlabs.org/go/Página_Principal" class="hrefCustomColor">Sugar Labs Chile</a></li>
```
Comment (quozl) [inline about-us.html]: Unintended whitespace change?

File: about-us.html [hunk 8]
```diff
@@ -224,24 +224,24 @@ <h2>LOCAL LABS</h2>
...
                             </ul>
-                            </p>  
+                            </p>
                         </div>
                     </div>
                 </div>
             </div>
-        </div> 
-    </div>    
+        </div>
```
Comment (quozl) [inline about-us.html]: Unintended whitespace change?

File: about-us.html [hunk 9]
```diff
@@ -224,24 +224,24 @@ <h2>LOCAL LABS</h2>
         <div class="row">
             <div class="col-md-12" >
                 <div class="row">
-                    <div class="col-md-8 col-md-push-2" > 
+                    <div class="col-md-8 col-md-push-2" >
```
Comment (quozl) [inline about-us.html]: Unintended whitespace change?

File: about-us.html [hunk 10]
```diff
@@ -224,24 +224,24 @@ <h2>LOCAL LABS</h2>
...
                         <div class="section-title text-center">
                             <h2>Roadmap</h2>
                             <p class="customParagraphStyle">The <a href="https://wiki.sugarlabs.org/go/Sugar_Labs/Roadmap" class="hrefCustomColor">Sugar Community roadmap</a> is used to guide our community efforts:
                             <ul>
                                 <li>Mission, Vision, Values</li>
-                                <li>Distribution</li> 
+                                <li>Distribution</li>
```
Comment (quozl) [inline about-us.html]: Unintended whitespace change?

File: about-us.html [hunk 11]
```diff
@@ -224,24 +224,24 @@ <h2>LOCAL LABS</h2>
...
                                 <li>Deployments</li>
                                 <li>Quality Assurance</li>
                                 <li>Infrastructure</li>
                             </ul>
-                            </p>  
+                            </p>
```
Comment (quozl) [inline about-us.html]: Unintended whitespace change?

File: about-us.html [hunk 12]
```diff
@@ -254,4 +254,3 @@ <h2>Roadmap</h2>
         </div>
     </div>
 </section>
-
```
Comment (quozl) [inline about-us.html]: Unintended whitespace change?

File: about-us.html [hunk 13]
```diff
@@ -201,11 +201,11 @@ <h2>Projects</h2>
             <div class="col-md-12" >
                 <div class="row">
                     <div class="col-md-8 col-md-push-2" >
-                        <h2>LOCAL LABS</h2> 
+                        <h2>LOCAL LABS</h2>
```
Comment (quozl) [inline about-us.html]: Unintended whitespace change?

File: about-us.html [hunk 14]
```diff
@@ -185,13 +185,13 @@ <h2>Projects</h2>
...
                             </ul>
-                            </p>  
+                            </p>
                         </div>
                     </div>
                 </div>
             </div>
-        </div> 
-    </div>    
+        </div>
```
Comment (quozl) [inline about-us.html]: Unintended whitespace change?

File: about-us.html [hunk 15]
```diff
@@ -185,13 +185,13 @@ <h2>Projects</h2>
                                 <li><a href="https://wiki.sugarlabs.org/go/Math4Team" class="hrefCustomColor">Math4 Project</a></li>
                                 <li><a href="https://wiki.sugarlabs.org/go/Summer_of_Code" class="hrefCustomColor">Summer of Code</a></li>
                             </ul>
-                            </p>  
+                            </p>
```
Comment (quozl) [inline about-us.html]: Unintended whitespace change?

**Comment by kushagra189** [comment]
@quozl  I am sorry about the whitespace changes. I am new to contributing and didn't think that such small changes could affect the code base largely. I will try to keep it in mind in future.
As far as the changes are concerned I am open to suggestion and would be glad to continue to work on whatever you suggest.

**Comment by quozl** [comment]
Thanks for your work so far.

I've reverted the permissions changed, merged with master (as master had changed since you began), and reverted the indentation changes.  The [resulting aggregate change](https://github.com/sugarlabs/www-sugarlabs/pull/197/files) is easier to understand;

* you changed the layout from three columns to one,
* you changed the test justification.

However, this isn't what https://github.com/sugarlabs/www-sugarlabs/issues/164 asked to do, and your screenshot doesn't match.  So you are proposing an alternate design?  Just want to make sure that's what you wanted to do before I merge it.

**Comment by kushagra189** [comment]
@quozl  yes. It was not exactly what was asked but it looks better in appearance.

**Comment by quozl** [comment]
Thanks.  Subjective.  So let's get back to the issue thread and gain consensus from others.

**Pull request closed without merging**